### PR TITLE
zip

### PR DESCRIPTION
**Pull request by return02**
implement zip

### CODE REVIEW

File: src/rdd/rdd.rs [hunk 1]
```diff
@@ -666,6 +667,23 @@ pub trait Rdd: RddBase + 'static {
...
+    ) -> SerArc<dyn Rdd<Item = (Self::Item, S)>>
+        where
+            Self: Clone,
+    {
+        SerArc::new(
+            ZippedPartitionsRdd::<Self::Item, S>::new(
+                Arc::new(
+                    self.clone()
+                ) as Arc<dyn Rdd<Item = Self::Item>>,
+                second.clone()
```
Comment (iduartgomez) [inline src/rdd/rdd.rs]: this second clone is probably unnecessary

Comment (return02) [reply src/rdd/rdd.rs]: > this second clone is probably unnecessary

check it later.

**Comment by rajasekarv** [comment]
It looks great @return02 . Thanks for the contribution

**Comment by rajasekarv** [comment]
closes #57

**Pull request merged**